requirements.txt

```diff
@@ -1 +1 @@
-Theano==0.7
+git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
```

Comment:
We will try to make a release this summer. But it isn't sure
Le 4 juil. 2015 09:31, "Sander Dieleman" <notifications@github.com> a
écrit :

> In requirements.txt
> <https://github.com/Lasagne/Lasagne/pull/320#discussion_r33887498>:
>
> > @@ -1 +1 @@
> > -Theano==0.7
> > +git+https://github.com/Theano/Theano.git@4dacb9e7d04d842d8ffe2c948dbc90722ae990d4#egg=Theano
>
> I see. Well, we should definitely consider it - I'm not strictly opposed
> to it. But I would like to hear as many of the core team members' input as
> possible (most notably @f0k <https://github.com/f0k> and @dnouri
> <https://github.com/dnouri> who I think will have more insight than me
> into the implications of this change).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Lasagne/Lasagne/pull/320/files#r33887498>.
>